apps/admin/src/components/form/submit-button.tsx

```
@@ -0,0 +1,24 @@
+
+type Props = {
+  label: string;
+};
+
+export const SubmitButton = ({ label }: Props) => {
+  const form = useFormContext();
+
+  return (
+    <form.Subscribe
+      selector={(state) => [state.isSubmitting, state.canSubmit]}
+      children={([isSubmitting, canSubmit]) => (
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)